src/shell/Shell.cpp

```diff
@@ -77,8 +77,10 @@ ValueRef* builtinPrint(ExecutionStateRef* state, ValueRef* thisValue, size_t arg
     if (argc >= 1) {
         if (argv[0]->isSymbol()) {
             puts(argv[0]->asSymbol()->symbolDescriptiveString()->toStdUTF8String().data());
+            state->context()->printDebugger(argv[0]->asSymbol()->symbolDescriptiveString());
```

Comment:
You could put `argv[0]->asSymbol()->symbolDescriptiveString()` into a local variable.